Cookbook/Technical-Documents/Localizations.md

```diff
@@ -0,0 +1,137 @@
+# How to deal with Localizations
+
+In project we can add a new Localization. But that is not in target level. Babylon needs to have a different sets of languages/locales per product.
+
+Babylon:
+  - British English
+
+BabylonKSA:
+  - Arabic
+
+NHS111:
+  - British English
+
+Telus:
+  - Canadian English
+  - Canadian French (soon)
+
+Babylon US and Ascension:
+  - American English
+  - American Spanish
+
+## Build Settings
+### Understanding `app_languages` variable
+Given that, we manage our languages in an array in Build Settings in each target.
+
+| Babylon UK | Babylon US | Telus |
+|--------|--------|--------|
+| `en.lproj/*` | `en-US.lproj/*` `es-US.lproj/*` | `en.lproj/*` |
+
+**I.E we control the languages for each target editing the `APP_LANGUAGES` under _User-Defined_**
+
+![Select the app target, Build Settings, User Defined](Assets/appLanguages-buildSettings-userDefined.png)
+
+### _Excluded Source File Names_ and _Included Source File Names_ under `Build option`
+In the _Excluded Source File Names_ we exclude any language (`.lproj/`) previously set:
+`*.lproj/*` 
+
+In the _Included Source File Names_ we include the languages (`.lproj/`) set in the above mentioned `app_languages` array:
+`${APP_LANGUAGES}` in Babylon US for example it is en-US.lproj/* es-US.lproj/*
+
+## How to add a new or changing an existing key/value
+### Add a new key/value
+In Lokalise, select the corresponding project. 
+Click on the `Add key ⌘K` button
+- `key` - Give a name for the key following the bread-crumb style. For example: `add_family_member_email_placeholder`, `biometric_touchid_primer_description`.
+    
+- `Base language value`: The actual string value corresponding to the first language in the list. Placeholders are supported with the `%@` where dynamic values are expected. 
+
+### Edit a key/value
+Search and select the key you'd like to update and change any language you need. The languages you may not have the values will be marked as `Not-verified` and the translators will take care.
+
+## How to add a new Locale
+- In Lokalise project, click on the `+` plus button beside other flags. Find the desired language on the list, and then add it.
+- In the iOS project, in the file `lokalise` (under the fastlane folder), add the language code (`en_US` for example) in the `langs` parameter for the desired target. `langs: 'es_US,en_US'` (comma separated without space). 
+
```

Comment:
I think there should be a third point here, similar-ish to the below:

```suggestion
- In Xcode - go to the Babylon Project, under localizations select the specific language you want to add to the project
<img width="674" alt="Screenshot 2020-01-07 at 14 40 16" src="https://user-images.githubusercontent.com/24831493/71903320-b794f900-315b-11ea-82c9-d6c0b947a7c7.png">

```

Reply:
we made this 👍

Reply:
Resolving this - @daniloaliberti fixed it in his latest commits